src/ashley/templates/partials/user_icon.html

```diff
@@ -6,4 +6,9 @@
     <span class="sr-only">{%trans 'Instructor'%}</span>
     </i>
 {% endif %}
-{% endspaceless %}
\ No newline at end of file
+{% if topic|is_user_administrator:user %}
+    <i class="icon_writer fas fa-award" aria-hidden="true" title="{%trans 'Administrator'%}">
```

Comment:
Would there be any interest in showing a different sign for an admin vs an instructor?

Reply:
I wondered as well...but none inspired me

Reply:
I've asked C&P, no need to do it.